src/ResourceManager/AzureBatch/Commands.Batch.Test/ScenarioTests/ScenarioTestHelpers.cs

```diff
@@ -507,7 +507,7 @@ public static void WaitForIdleComputeNode(BatchController controller, BatchAccou
                 Select = "id,state"
             };
 
-            DateTime timeout = DateTime.Now.AddMinutes(5);
+            DateTime timeout = DateTime.Now.AddMinutes(10);
```

Comment:
Can we change this to 3 hours?

Reply:
I wanted to see where the tests failed in CI, and then fix up all those places, as I am not sure this is the only place we wait.

I will amend the PR to wait a longer time once I've seen the initial results though, yes.